sqlglot/dialects/spark.py

```diff
@@ -32,11 +32,11 @@ def _unix_to_time(self, expression):
     timestamp = self.sql(expression, "this")
     if not scale:
         return f"FROM_UNIXTIME({timestamp})"
-    if scale.name == "0":
+    if scale == exp.UnixToTime.SECONDS:
```

Comment:
on line 31 change it to epression.text("scale")

Reply:
fixed